File: ws/gavilanez/u2/ws21Inherence/Zoo/src/main/java/ec/edu/espe/zoo/model/Shark.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Kenny Gavilanez, Dev Dynasty, DCCO-ESPE
 */
public class Shark extends Fish{

    public Shark(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }
    
    
    @Override
    public void Feed(int food){
        System.out.println("feeding to Shark");
    }
    
}

File: hw/ccedeno/u2/HW18ZooInPythonJavaScript/ZooInJavaScript/ec.edu.espe.Zoo.model/Shark.js
const Fish = require('../ec.edu.espe.Zoo.model/Fish');

class Shark extends Fish {
  constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
    super(id, description, dateOnborn, gender, numberOfBones, cage);
  }

  feed(food) {
    console.log("Feeding a Shark");
  }
}

module.exports = Shark;
